docs/OV_Runtime_UG/deployment/deployment-manager-tool.md

```diff
@@ -185,21 +186,22 @@ To deploy the OpenVINO Runtime components from the development machine to the ta
 
 @endsphinxdirective
 
+
   The package is unpacked to the destination directory and the following files and subdirectories are created:
 
-   * `setupvars.sh` — Copy of `setupvars.sh`
-   * `runtime` — Contains the OpenVINO runtime binary files.
-   * `install_dependencies` — Snapshot of the `install_dependencies` directory from the OpenVINO installation directory.
-   * `<user_data>` — The directory with the user data (IRs, datasets, etc.) you specified while configuring the package.
+   * `setupvars.sh` — a copy of `setupvars.sh`,
+   * `runtime` — contains the OpenVINO runtime binary files,
+   * `install_dependencies` — a snapshot of the `install_dependencies` directory from the OpenVINO installation directory,
+   * `<user_data>` — the directory with the user data (IRs, datasets, etc.) specified while configuring the package.
```

Comment:
```suggestion
   * `<user_data>` — the directory with the user data (OpenVINO IRs, datasets, etc.) you specified while configuring the package.
```

Reply:
Shouldn't we include info about model as well? 

```suggestion
   * `<user_data>` — the directory with the user data (OpenVINO IR, model, dataset, etc.) specified while configuring the package.
```
See your suggestion in line 146 for reference.

Reply:
Aha, yes we can add models. I use plurals here because the word "files" is not following "user data" this time. Maybe we should use the same words as they are saying the same thing. :)

Reply:
Ok. So l guess it won't hurt to add models, as long as it refers to the same 'user data' collective :)